### DISCUSSION TITLE
PremBunsen3D crashes

### DISCUSSION THREAD

**Discussion by sjlienge** [AMReX-Combustion/PeleLMeX#309] [Q&A] [answered]
Hello,

The PremBunsen3D case crashes after printing the Build infos with SIGABRT. The same also applies for the PremBunsen2D. Is it known what the issue is? The source files seem alright to me. I would like to use the case to understand the setup of burner flames and setup a calculation of the Cambridge burner.

Thank you in advance already!

**Comment by baperry2** [comment, marked as answer]
It looks like these cases don't use EB, but are being compiled with EB capability. You can either recompile with `USE_EB = FALSE` in your GNUmakefile or set `eb2.geom_type = all_regular` in your input file.

**Reply by sjlienge** [reply]
Thank you so much, that worked and the case runs now!
